### PR TITLE
Tr 2339 - Added getting started guide for Engagement Tracking.

### PR DESCRIPTION
**Pull request by ndurant-sp**
### Support Docs Update Checklist

Note: when you open a pull request, your changes are published to https://staging.sparkpost.com/docs.

Please take a moment to check each of these steps for your pull request:

- [ ] Give your pull request a meaningful name.
- [ ] Include a description of your changes in the pull request.
- [ ] _SparkPost staff only_: Check that your article appears [here](https://staging.sparkpost.com/docs) and looks correct.
- [ ] Check the links in your article.
- [ ] Use lowercase filenames.

### CODE REVIEW

File: support/getting-started/engagement-tracking.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+With SparkPost, there are a multitide of ways to configure engagement tracking with our customers.  This guide will outline some of the initial options, and provide references to other guides for configuring custom tracking domains and different ways to use **HTTPS** with your custom tracking domains.
+
+## Getting Started
+Customers can view their engagement through the use of SparkPosts Events.  These events can be searched and viewed in many different ways.  They can be viewed within [SparkPost Signals](https://www.sparkpost.com/docs/reporting/signals-analytics/), queried and searched via the [Events API](https://developers.sparkpost.com/api/events/) or sent to your own environments through the use of [Event Webhooks](https://developers.sparkpost.com/api/webhooks/) to name a few.
+
+
+### Notes on Click Tracking
+SparkPost click tracking links are highly dependent on the size of the original link supplied within the REST Transmission or SMTP request.  If there are concerns over link length within an email with SparkPost's click tracking enabled, first verify the original link length.
+
+> Link sizes of over **4096** bytes will be rejected on redirection
```
Comment (dremsberg-msys) [inline support/getting-started/engagement-tracking.md:15]: Do we know approximately how much we add to the original link typically -to give them a hint.  Or is that two difficult to quantify?

File: support/getting-started/engagement-tracking.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## REST Transmissions with Engagement Tracking
+Transmissions sent via the SparkPost Transmissions API have engagement tracking on by default for all customers.  This includes a top tracking pixel, and bottom open pixel, and click tracking on all links within the email content.  These options can be configured via [Transmissions API](https://developers.sparkpost.com/api/transmissions/) level options.  For example, if you wish to use open tracking, but not click tracking, you can set the following options in your JSON payload for a REST transmission
+
```
Comment (dremsberg-msys) [inline support/getting-started/engagement-tracking.md:19]: Stored templates also have per template click and open tracking options.  They may also be overridden by transmission options.

File: support/getting-started/engagement-tracking.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Notes on Click Tracking
+SparkPost click tracking links are highly dependent on the size of the original link supplied within the REST Transmission or SMTP request.  If there are concerns over link length within an email with SparkPost's click tracking enabled, first verify the original link length.
+
+> Link sizes of over **4096** bytes will be rejected on redirection
+
+## REST Transmissions with Engagement Tracking
+Transmissions sent via the SparkPost Transmissions API have engagement tracking on by default for all customers.  This includes a top open pixel, and bottom open pixel, and click tracking on all links within the email content.  These options can be configured via [Transmissions API](https://developers.sparkpost.com/api/transmissions/) level options.  For example, if you wish to use open tracking, but not click tracking, you can set the following options in your JSON payload for a REST transmission
```
Comment (ischer) [inline support/getting-started/engagement-tracking.md:18]: `This includes a top open pixel, and bottom open pixel, and click tracking on all links within the email content.` - Could probably remove the `and` before `bottom open pixel`. NBD, though.

Line below (29) has a trailing comma that could probably be removed, as well.
```
"ip_pool": "my_ip_pool",
```

File: support/getting-started/engagement-tracking.md (hunk 1 above)
Comment (ndurant-sp) [reply support/getting-started/engagement-tracking.md:15]: I guess we could "estimate" just based on some testing, but not sure how much we want to give away.

File: support/getting-started/engagement-tracking.md (hunk 2 above)
Comment (ndurant-sp) [reply support/getting-started/engagement-tracking.md:19]: Oh good one, thanks.  Will add.

File: support/getting-started/engagement-tracking.md (hunk 3 above)
Comment (ndurant-sp) [reply support/getting-started/engagement-tracking.md:18]: Ha yep thanks.

**Pull request merged**